symposion/reviews/views.py

```diff
@@ -93,13 +94,21 @@ def review_section(request, section_slug, assigned=False):
     queryset = ProposalBase.objects.filter(kind__section=section)
 
     if request.method == "POST" and can_manage:
-        pk = request.POST['pk']
-        status = request.POST['status']
-        base_obj = queryset.get(pk=pk)
-        p_type = section_slug.rstrip('s').replace('-', '')
-        proposal = getattr(base_obj, 'pycon%sproposal' % p_type)
-        proposal.overall_status = status
-        proposal.save()
+        pk_string = request.POST['pk']
```

Comment:
I think I'd rename this input from `pk` to `pks`, both to reflect the change in syntax and to make sure we haven't missed updating any references to the old syntax.